tsconfig.json

```diff
@@ -14,23 +14,26 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "incremental": true,
-    "types": ["react", "node"],
+    "types": ["react", "jest", "node"],
     "plugins": [
       {
-        "name": "next"
-      }
+        "name": "next",
+      },
     ],
     "paths": {
-      "@/*": ["./src/*"]
-    }
+      "@/*": ["./src/*"],
+    },
   },
   "include": [
     "next-env.d.ts",
     "**/*.ts",
     "**/*.tsx",
     ".next/types/**/*.ts",
     "next.config.js",
-    "postcss.config.js"
+    "postcss.config.js",
+    "./jest.config.js",
+    "./jest.setup.js",
+    ".lint-staged.config.js", "src/utils/cookieUtils.ts",
   ],
```

Comment:
Why have you added this back?

Reply:
conflict resolution error! There's been a mix-up.